docs/content/components/box.md

```diff
@@ -531,10 +531,10 @@ You can put forms in boxes. Often form submission buttons are aligned to the bot
   </div>
   <form>
     <div class="Box-body">
-      <dl class="form-group">
-        <dt><label>Example label</label></dt>
-        <dd><input class="form-control" type="text" /></dd>
-      </dl>
+      <div class="form-group">
+        <div class="form-group-header"><label>Example label</label></div>
+        <div class="form-group-body"><input class="form-control" type="text" /></div>
```

Comment:
FWIW ending slashes were meant for XML, never HTML. In `github/github` we have a linter against them.

```suggestion
        <div class="form-group-body"><input class="form-control" type="text"></div>
```